velox/docs/functions/spark/decimal.rst

```diff
@@ -43,6 +57,8 @@ precision and scale are adjusted.
     precision = 38
     scale = max(38 - (p - s), min(s, 6))
 
+Caps p and s at 38 when not allowing precision loss.
```

Comment:
Perhaps move below part here to illustrate how precision and scale are adjusted when allowPrecisionLoss is false.

> If ``indDig + decDig`` is more than 38:
::
    p = 38
    s = decDig - (intDig + decDig - 38) / 2 - 1